lib/Raven/Client.php

```diff
@@ -414,6 +426,12 @@ public function captureMessage($message, $params=array(), $data=array(),
      */
     public function captureException($exception, $data=null, $logger=null, $vars=null)
     {
+        //avoid errors other than specific path
+        $exception_file = $exception->getFile();
+        if (isset($this->strict_error_capture_path) && !empty($this->strict_error_capture_path) && stripos($exception_file, $this->strict_error_capture_path) === false) {
```

Comment:
You can leave out the `isset` check and leave only the `empty`.

See: http://stackoverflow.com/a/4559976/1580028